### PR TITLE
feat: restrict automerge schedule to business hours

### PR DESCRIPTION
**Pull request by phillip-kil**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Does not affect documentation or it has been added or updated.

This PR limits the auto merge schedule to restricted business hours during on only weekdays, so we can have peace of mind that no runtime breaking issues are merged.

This is a follow-up to the [Dependency update causing runtime errors](https://www.notion.so/ridedott/Dependency-update-causing-runtime-errors-20cc8552b6bc494d962b0cdd8402d220) post mortem.

Resolves #

### CODE REVIEW

**Review by reyawn — APPROVED** [review]

**Comment by markelog** [comment]
This bump should be major release, so if you merge it – please bump the version with it

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ The default configuration is currently set to:
 
 ```json
 {
+  "automergeSchedule": ["after 9am and before 4pm every weekday"],
```
Comment (markelog) [inline README.md:96]: I don't how everyone else are, but 80% of partner integration members is asleep before 10 :), I would put it after 1 or even 2 pm.

This is more controversial I suppose – as a default, I would also put it as once a week, and maybe add exception for `firestore-schemas` to update it more frequently.

**Review by markelog — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (phillip-kil) [reply README.md:96]: 1. I get your point, I think when we look at "default" business hours I think of 09:00-17:00, so I think we use that timeframe minus some buffer on both ends. (I'll update the schedule to be from 10:00-16:00.) And for your specific case I'd prefer an override in the repositories your team maintains tbh.
1. To me the main idea of the `automergeSchedule` is that those are only happening while we are at work and not bothering on call engineers with runtime issues due to automatically merged dependency updates like we've seen here https://www.notion.so/ridedott/Dependency-update-causing-runtime-errors-20cc8552b6bc494d962b0cdd8402d220.  In general I'd abstain from having exceptions to this, even though I agree with those `firestore-schema` PRs usually being safe in this regard.

Comment (phillip-kil) [reply README.md]: @markelog can you have a look at this again?

**Review by reyawn — DISMISSED** [review]

**Review by mikeljh — DISMISSED** [review]

**Review by dunyakirkali — DISMISSED** [review]
LGTM

**Review by dunyakirkali — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (phillip-kil) [reply README.md]: @markelog pinging you here again, thanks for having another look!

**Review by dunyakirkali — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (markelog) [reply README.md]: Sorry for delay, a bit of vacation and after vacation hassle got me busy

**Review by markelog — APPROVED** [review]

**Pull request merged**